packages/kit-bg/src/services/ServiceHistory.ts

```diff
@@ -22,6 +25,22 @@ class ServiceHistory extends ServiceBase {
 
   @backgroundMethod()
   public async fetchAccountHistory(params: IFetchAccountHistoryParams) {
+    const { accountId, networkId, tokenIdOnNetwork } = params;
+    const onChainHistoryTxs = await this.fetchAccountOnChainHistory(params);
+
+    await simpleDb.localHistory.updateLocalHistoryPendingTxs(onChainHistoryTxs);
```

Comment:
```suggestion
    await this.backgroundApi.simpleDb.localHistory.updateLocalHistoryPendingTxs(onChainHistoryTxs);
```

simpledb 已经挂载在 backgroundApi 上了，不需要单独 import，其他地方也一样

Reply:
fixed